config.sh

```diff
@@ -52,8 +81,10 @@ function build_brotli {
 function pre_build {
     # Any stuff that you need to do before you start building the wheels
     # Runs in the root directory of this repository.
-    curl -fsSL -o pillow-depends-main.zip https://github.com/python-pillow/pillow-depends/archive/main.zip
-    untar pillow-depends-main.zip
+    if [ ! -d pillow-depends-main ]; then
+        curl -fsSL -o pillow-depends-main.zip https://github.com/python-pillow/pillow-depends/archive/main.zip
+        untar pillow-depends-main.zip
+    fi
```

Comment:
The build is run twice for universal2, once for x86_64 and once for arm64, and pillow-depends only needs to be downloaded once.